src/PowerShellEditorServices/Services/TextDocument/Handlers/DocumentSymbolHandler.cs

```diff
@@ -50,10 +50,7 @@ public override async Task<SymbolInformationOrDocumentSymbolContainer> Handle(Do
             ScriptFile scriptFile = _workspaceService.GetFile(request.TextDocument.Uri);
 
             IEnumerable<SymbolReference> foundSymbols = ProvideDocumentSymbols(scriptFile);
-            if (foundSymbols is null)
-            {
-                return null;
-            }
+            foundSymbols = foundSymbols.Concat(RegionVisitor.GetRegionsInDocument(scriptFile));
 
```

Comment:
I didn't see a reason to keep the null-check. The local method `ProvideDocumentSymbols()` always returns a list, right?